plaidml/BUILD

```diff
@@ -394,6 +410,12 @@ plaidml_cc_binary(
 
         # Use an explicit linker script.
         "-Wl,$(location :plaidml.lds)",
+
+        # Link static C++ dependencies.
+        "-static-libgcc",
+        "-Wl,-Bstatic",
+        "-lstdc++",
+        "-Wl,-Bdynamic",
```

Comment:
This seems duplicated from the base linker flags, is it required here?

Reply:
Nope, no longer required.  Removed.